### DISCUSSION TITLE
Difference between Microsoft.Extensions.Caching.Memory & SystemRuntimeCaching

### DISCUSSION THREAD

**Discussion by Jogai** [MichaCo/CacheManager#344] [Q&A] [answered]
I have both in my project anyways, which brings me to the question, does one have an advantage over the other?

Second question, does CacheManager 1.2 work with Microsoft.Extensions.Caching.Memory 2.2?

**Comment by MichaConrad** [comment, marked as answer]
Those two caches have very different implementations. The Extesions.Caching one uses ConcurrentDictionary, the SystemRuntimeCaching uses a shared memory buckets per Core as far as I know.

The Eviction and other configuration options are different two.
Depends on the platform and use cases I guess.

Regarding the 2nd questions, it should.
